Decode/PerfItemValue.cs

```diff
@@ -1000,6 +1057,44 @@ public string GetString()
             return sb;
         }
 
+        /// <summary>
+        /// Converts this to a JSON value and appends it to sb. Returns sb.
+        /// <br/>
+        /// Cannot be used for struct or complex array.
+        /// <br/>
+        /// For example:
+        /// <list type="bullet"><item>
+        /// If the value is a scalar decimal integer or a finite float, appends a JSON number
+        /// like <c>123</c> or <c>-123.456</c>.
+        /// </item><item>
+        /// If the value is a scalar boolean, appends a bool <c>false</c> (for 0), <c>true</c> (for 1),
+        /// a string like <c>"BOOL(-123)"</c> if convertOptions has BoolOutOfRangeAsString, or a number
+        /// like <c>-123</c> otherwise.
+        /// </item><item>
+        /// If the value is a scalar string, appends a JSON-escaped string like <c>"abc\nxyz"</c>.
+        /// </item><item>
+        /// If the value is a simple array, appends a JSON array like <c>[ 1, 2, 4, 8 ]</c>.
+        /// </item></list>
+        /// </summary>
+        /// <exception cref="NotSupportedException">
+        /// Encoding is not recognized.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// Item is a struct.
+        /// <br/>
+        /// OR
+        /// <br/>
+        /// Item is an array and encoding is a complex type (variable-length or struct).
+        /// </exception>
+        public StringBuilder AppendJsonTo(
```

Comment:
Note: Now that the metadata tracks whether the value is a scalar, we can expose this method that should have been there from the start.